### ISSUE TITLE
Focus outline contrast issues

### ISSUE THREAD

**Issue by petarbk** [vmware/clarity#4686]
## Description

If you're using the dark theme with Firefox there are contrast issues with the focus outline of Signpost and Tabs components. The outline is black on a dark background which leads to contrast lower than 3:1 ratio.

## How to reproduce

You can see the issue here: https://stackblitz.com/edit/clarity-v3-dark-theme-focus-outline-issues?file=src%2Fapp%2Fapp.component.html

NOTE: Use Firefox to open the link and focus the elements using the keyboard. If you toggle the
focus using developer tools, the outline is white and it seems there is no problem. 

## Expected behavior

The default browser behavior of Firefox is to set the outline-color to the color of the element. So if the contrast ratio of the chosen color is good, the outline-color will be good too. 
Currently, this is the behavior of Clarity's flat buttons.

## Versions

**App**

- Clarity: 3

**Device:**

- Type: MacBook
- OS: MacOS Catalina 10.15.4
- Browser: Firefox
- Version: 76.0.1 (64 bit)

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
